getting_started/README.md

```diff
@@ -0,0 +1,73 @@
+# Getting started
+
+Hi and welcome to Crystal's Reference Book!
+
+First we are going to check if we have [installed the language](https://crystal-lang.org/install/) correctly so that we may try all the examples listed in this book.
```

Comment:
The text shouldn't assume readers have already installed the language, might have come here first. Let's guide them to the installation instructions.

```suggestion
First let's make sure to [install the compiler](https://crystal-lang.org/install/) correctly so that we may try all the examples listed in this book.
```